examples/example4.json.txt

```diff
@@ -1,92 +1,92 @@
+0d69f35a89bff35ea5aaf308ddb471a9b47b8b0b1524f4e80194a72fa7153bc78ad67f
+5647295990195893a967'
+            ])>>
         ]
-    ]),
-    / manifest / 3:bstr .cbor ({
+    ]>>,
+    / manifest / 3:<<{
         / manifest-version / 1:1,
         / manifest-sequence-number / 2:4,
-        / common / 3:bstr .cbor ({
+        / common / 3:<<{
             / components / 2:[
                 [h'00'] ,
                 [h'02'] ,
                 [h'01']
             ],
-            / common-sequence / 4:bstr .cbor ([
+            / common-sequence / 4:<<[
                 / directive-set-component-index / 12,0 ,
                 / directive-override-parameters / 20,{
                     / vendor-id /
 1:h'fa6b4a53d5ad5fdfbe9de663e4d41ffe' / fa6b4a53-d5ad-5fdf-
 be9d-e663e4d41ffe /,
                     / class-id / 2:h'1492af1425695e48bf429b2d51f2ab45'
 / 1492af14-2569-5e48-bf42-9b2d51f2ab45 /,
-                    / image-digest / 3:bstr .cbor ([
+                    / image-digest / 3:<<[
                         / algorithm-id / 2 / "sha256" /,
                         / digest-bytes /
 h'00112233445566778899aabbccddeeff0123456789abcdeffedcba9876543210'
-                    ]),
+                    ]>>,
                     / image-size / 14:34768,
                 } ,
                 / condition-vendor-identifier / 1,15 ,
                 / condition-class-identifier / 2,15
-            ]),
-        }),
-        / payload-fetch / 8:bstr .cbor ([
+            ]>>,
+        }>>,
+        / payload-fetch / 8:<<[
             / directive-set-component-index / 12,1 ,
             / directive-set-parameters / 19,{
                 / uri / 21:'http://example.com/file.bin',
             } ,
             / directive-fetch / 21,2 ,
             / condition-image-match / 3,15
-        ]),
-        / install / 9:bstr .cbor ([
+        ]>>,
+        / install / 9:<<[
             / directive-set-component-index / 12,0 ,
             / directive-set-parameters / 19,{
                 / source-component / 22:1 / [h'02'] /,
             } ,
             / directive-copy / 22,2 ,
             / condition-image-match / 3,15
-        ]),
-        / validate / 10:bstr .cbor ([
+        ]>>,
+        / validate / 10:<<[
             / directive-set-component-index / 12,0 ,
             / condition-image-match / 3,15
-        ]),
-        / load / 11:bstr .cbor ([
+        ]>>,
+        / load / 11:<<[
             / directive-set-component-index / 12,2 ,
             / directive-set-parameters / 19,{
-                / image-digest / 3:bstr .cbor ([
+                / image-digest / 3:<<[
                     / algorithm-id / 2 / "sha256" /,
                     / digest-bytes /
 h'0123456789abcdeffedcba987654321000112233445566778899aabbccddeeff'
-                ]),
+                ]>>,
                 / image-size / 14:76834,
                 / source-component / 22:0 / [h'00'] /,
                 / compression-info / 19:1 / "gzip" /,
```

Comment:
Something like this?
```suggestion
                / compression-info / 19:<< { / 1:1 / "gzip" /} >>,
```

Reply:
You're right, looks like the compression parameter isn't being generated correctly.